packages/frontity-org-theme/src/processors/image-frame.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { css, styled } from "frontity";
+import { Processor } from "@frontity/html2react/types";
+
+const Dot = styled.span`
+  display: inline-block;
+  width: 9px;
+  height: 9px;
+  margin-left: 5px;
+  border-radius: 100%;
+  background-color: rgba(15, 28, 100, 0.2);
+`;
+
+const Top = () => (
+  <div
+    css={css`
+      height: 32px;
+      padding-left: 10px;
+    `}
+  >
+    <Dot />
+    <Dot />
+    <Dot />
+  </div>
+);
+
+const imageFrame: Processor<React.HTMLProps<HTMLElement>> = {
+  name: "border-radius",
```

Comment:
Not a good name for this processor, I guess. 😅

PS: I actually saw that some of them were wrong too so I changed them and pushed it to `dev`.